dev-haskell/beam-automigrate/beam-automigrate-0.1.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="DB migration library for beam, targeting Postgres"
+HOMEPAGE="https://hackage.haskell.org/package/beam-automigrate"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="ghcipretty integration-tests werror examples"
+
+PATCHES=( "${FILESDIR}/${P}-add-examples-flag.patch" )
```

Comment:
Why is this patch needed? It looks like there are [proper `test-suite`][1] stanzas in the cabal file.

[1]: https://github.com/obsidiansystems/beam-automigrate/blob/v0.1.2.0/beam-automigrate.cabal#L102

Reply:
There are a number of example executables that are defined in that cabal file, like this one:
https://github.com/obsidiansystems/beam-automigrate/blob/develop/beam-automigrate.cabal#L127

By default, they are built and installed, but I don't think most people would want them. So, I added an `examples` USE flag / cabal flag. That patch adds the functionality to the cabal file so those example executables can be toggled with that flag.